Geometry/TrackerNumberingBuilder/bin/stubs/GeometricDetLoader.h

```diff
@@ -24,6 +24,7 @@ class GeometricDetLoader : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
 
 private:
   void putOne(const GeometricDet* gd, PGeometricDet* pgd, int lev);
+  bool fromDD4hep;
```

Comment:
My question is certainly beyond this PR, but why is a plugin defined in `bin`?

Reply:
@makortel I am not sure that anyone still in CMS would know the ancient history of this plugin. It has probably been working like this for many years.

Reply:
I suspect the plugin is not built at all. Also `git grep GeometricDetLoader` shows nothing beyond the definition of the class. If it really is unused, I'd suggest to remove it (I can open an issue about it if people prefer).